src/Maestro/SubscriptionActorService/BatchedPullRequestActorImplementation.cs

```diff
@@ -25,6 +26,14 @@ internal class BatchedPullRequestActorImplementation : PullRequestActorImplement
     private readonly ActorId _id;
     private readonly BuildAssetRegistryContext _context;
 
+    /// <param name="id">
+    ///     The actor id for this actor.
```

Comment:
I agree this is not great but also not sure we should start improving unrelated things in this PR. It's kind of already bloated.

This is a rabbit hole a bit because for instance `Target` is a property but calls parsing inside and we call the parsing twice because it's not obvious it's doing that etc.